src/ReactiveSocket.h

```diff
@@ -32,6 +32,8 @@ class KeepaliveTimer {
 
 using CloseListener = std::function<void(ReactiveSocket&)>;
 
+using ResumeSocketListener = std::function<bool(ReactiveSocket& newSocket, const ResumeIdentificationToken& token)>;
```

Comment:
I think it would be better to let the application track the tokens and associations. So, that means it would be external.